### DISCUSSION TITLE
How revoke other user session?

### DISCUSSION THREAD

**Discussion by cesarve77** [blitz-js/blitz#2749] [Q&A] [answered]
Administrator wants to downgrade access to a user.

Thanks for your time.

I am very new with blitz and I like it.

**Comment by MrLeebo** [comment, marked as answer]
You can delete all of the records for that user's ID in the Session table 👍

**Comment by cesarve77** [comment]
Thabks for your quick answer.

I don't need to delete tokens?

**Reply by MrLeebo** [reply]
No, tokens are used for stuff like resetting a forgot password, or email confirmation. It would be super frustrating if that user was in the middle of one of those and it suddenly stopped working since they wouldn't know why, and having fewer permissions in the app shouldn't mean you're not allowed to reset your password or change your email address

**Comment by cesarve77** [comment]
Thanks so much
